src/Data/Quaternion/Rotation.purs

```diff
@@ -107,3 +112,24 @@ act (Rotation p) v =
   unsafePartial $ case Vec.toArray v of
     [x, y, z] ->
       vectorPart (Quaternion zero x y z `conjugateBy` p)
+
+normalize :: Rotation Number -> Rotation Number
```

Comment:
I would end pulling from the new state of your master - if you just push to master it'll do the right thing. :)